### PR TITLE
Update flowchart to show that indels and fusions are implemented (resolves #258)

### LINKED ISSUE #258: Update flowchart
indels and fusions are implemented

### PR DESCRIPTION
**Pull request by swp930**

### CODE REVIEW

**Comment by swp930** [comment]
resolves #258 

Update flowchart to show that indels and fusions are implemented

**Comment by arkal** [comment]
Jenkins, test this please.

File: Flowchart.txt [hunk 1]
```diff
@@ -65,7 +65,7 @@
      9 = Delete Fastqs to cleanup space for future jobs
     10 = Merge PHLAT outputs (PYTHON SCRIPT)
     11 = Calculate Gene Expression (RSEM)
-    12 = Identify Fusion Genes (CURRENTLY NOT IMPLEMENTED)
+    12 = Identify Fusion Genes
```
Comment (arkal) [inline Flowchart.txt]: 12 = Identify Fusion Genes (Star Fusion + Fusion Inspector)

File: Flowchart.txt [hunk 2]
```diff
@@ -75,7 +75,7 @@
     19 = Merge Mutation Calls 2 (Mutect)
     20 = Merge Mutation Calls 3 (MuSE)
     21 = Merge Mutation Calls 4 (SomaticSniper)
-    22 = INDEL Calling (CURRENTLY NOT IMPLEMENTED)
+    22 = INDEL Calling
```
Comment (arkal) [inline Flowchart.txt]: 22 = INDEL Calling (Strelka)

**Pull request merged**